pyproject.toml

```diff
@@ -1,133 +1,5 @@
-[build-system]
```

Comment:
@abdeslamads this is very strange to almost empty pyproject.toml ^^
cc @echoix 

Are you up to date with main branch ?